test/sequential/feeLock.API.config.test.ts

```diff
@@ -104,7 +108,7 @@ test("gasless- GIVEN a feeLock WHEN periodLength and feeLockAmount are set THEN
       JSON.stringify(await api?.query.feeLock.feeLockMetadata())
     ).periodLength.toString()
   );
-  const lastFeeLockAmount = new BN(
+  const lastFeeLockAmount = stringToBN(
```

Comment:
Should we use here `getFeeLockMetadata`?

Reply:
yup, makes sense. will fix !